### PR TITLE
antlr 4.5.1

### PR DESCRIPTION
**Pull request by thii**

### CODE REVIEW

File: Library/Formula/antlr.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 class Antlr < Formula
   desc "ANTLR: ANother Tool for Language Recognition"
   homepage "http://www.antlr.org/"
-  url "http://www.antlr.org/download/antlr-4.5-complete.jar"
-  sha1 "d73369e018ca99c6d9ec609473395b8257d783d3"
+  url "http://www.antlr.org/download/antlr-4.5.1-complete.jar"
+  sha1 "19a08f261dc2b7e1bc58c8f1038f766dccade74c"
```
Comment (mikemcquaid) [inline Library/Formula/antlr.rb]: Would you mind changing this to a `sha256`? Thanks!

**Comment by mikemcquaid** [comment]
The test is failing: http://bot.brew.sh/job/Homebrew%20Pull%20Requests/29892/version=yosemite/testReport/junit/brew-test-bot/yosemite/test_antlr/

Does `brew test antlr` fail for you locally?

**Comment by dunn** [comment]
I forgot about this PR and pulled an update to `sha256`, so this just needs to be rebased now.

**Comment by DomT4** [comment]
Closing in favour of #42079.

**Pull request closed without merging**